upgrade-tests/src/test/java/io/syndesis/qe/UpgradeSteps.java

```diff
@@ -37,8 +37,8 @@
 @Slf4j
 public class UpgradeSteps {
     private static final String RELEASED_OPERATOR_IMAGE = "registry.redhat.io/fuse7/fuse-online-operator";
-    private static final String UPSTREAM_OPERATOR_IMAGE = "syndesis/syndesis-operator";
-    private static final String DOCKER_HUB_SYNDESIS_TAGS_URL = "https://hub.docker.com/v2/repositories/syndesis/syndesis-server/tags/?page_size=100";
+    private static final String UPSTREAM_OPERATOR_IMAGE = "quay.io/syndesis/syndesis-operator";
```

Comment:
can you please create a constant for the docker registry, so that we won't need to change it on 3 different places? (for 99% we won't change it, but you never know :grin:)